packages/kbn-legacy-logging/src/rotate/log_rotator.ts

```diff
@@ -351,22 +350,14 @@ export class LogRotator {
   }
 
   _sendReloadLogConfigSignal() {
-    if (isMaster) {
-      (process as NodeJS.EventEmitter).emit('SIGHUP');
+    if (!process.env.isDevCliChild || !process.send) {
+      process.emit('SIGHUP', 'SIGHUP');
```

Comment:
Why are there two arguments to this function? It appears the second argument is supposed to be a socket or server?

Reply:
That's what `@types/node` calls for, [signal listeners](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/v12/globals.d.ts#L751) get the signal as their first argument and the [emit type](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/v12/globals.d.ts#L1005) reflects that.